packages/cli/test/simulation/simulation.test.ts

```diff
@@ -62,14 +62,13 @@ for (const {beaconNodes, validatorClients, validatorsPerClient} of nodeCases) {
       validatorsPerClient,
       altairEpoch,
       // TODO: Use extra delay until env.clock is based on absolute time
-      genesisSlotsDelay: (SLOTS_PER_EPOCH * runTill + 50) * testCases + 30,
+      genesisSlotsDelay: SLOTS_PER_EPOCH * 2,
       bellatrixEpoch,
       logFilesDir: join(logFilesDir, testIdStr),
     });
-    testCases += 1;
 
     describe(`simulation test - ${testIdStr}`, function () {
-      this.timeout("5m");
+      this.timeout("10m");
```

Comment:
Can you add an issue to compute this timeout in a precise number that makes sense given the test requirements? i.e. expected duration time + 10%

Reply:
Added it as task to the main issue.